lib/common/util.js

```diff
@@ -124,7 +124,7 @@ util.arrayize = arrayize;
  */
 function format(template, args) {
   return template.replace(/{([^}]*)}/g, function(match, key) {
-    return args[key] || match;
+    return key in args ? args[key] : match;
```

Comment:
Am curious, how come this change? Doesn't the single lookup and indexing translate directly and in less steps to the intention of the method?